### PR TITLE
added files for debian packaging

### PR DESCRIPTION
**Pull request by unDocUMeantIt**
- the directory debian was added
  - it can be used for proper packaging, building the binary package
    seafile (desktop applet)
  - this is what enabled the PPA https://launchpad.net/~m.eik/+archive/ubuntu/seafile-experimental

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Source: seafile-gui
+Section: net
+Priority: extra
+Maintainer: m.eik michalke <meik.michalke@hhu.de>
+Build-Depends:
+    debhelper (>= 7),
+    dh-python,
+    python,
```
Comment (lins05) [inline debian/control]: `Seafile-client` has no python modules, so I guess we can remove `dh-python` and `python` from the build deps?

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    autotools-dev,
```
Comment (lins05) [inline debian/control]: I guess `autotools-dev` is also not required since we're using cmake here.

File: debian/rules [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+%:
+	dh $@ --with python2 --builddirectory=build
```
Comment (lins05) [inline debian/rules]: ditto for python

File: debian/rules [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+override_dh_auto_configure:
+	export QT_SELECT=5
+	dh_auto_configure -- -DUSE_QT5=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_SHIBBOLETH_SUPPORT=ON
```
Comment (lins05) [inline debian/rules]: `USE_QT5` has been removed for a while (we dropped qt4 and only supports qt5 since 5.1.0).

File: debian/control (hunk 1 above)
Comment (unDocUMeantIt) [inline debian/control]: aren't the two python scripts in the scripts directory (build_helper.py and build.py) used for the build process?

Comment (lins05) [inline debian/control]: They're only used in building the Mac OSX package.

Comment (unDocUMeantIt) [inline debian/control]: done.

File: debian/control (hunk 2 above)
Comment (unDocUMeantIt) [inline debian/control]: done.

File: debian/rules (hunk 3 above)
Comment (unDocUMeantIt) [inline debian/rules]: done.

File: debian/rules (hunk 4 above)
Comment (unDocUMeantIt) [inline debian/rules]: done.

**Pull request merged**